test/lib/AllowCallbacks.sol

```diff
@@ -5,19 +5,20 @@ import "quark-core/src/QuarkScript.sol";
 import "quark-core/src/QuarkWallet.sol";
 
 contract AllowCallbacks is QuarkScript {
-    // TODO: Uncomment when replay tokens are supported
-    // function run(address callbackAddress) public {
-    //     QuarkWallet self = QuarkWallet(payable(address(this)));
-    //     self.nonceManager().write(self.CALLBACK_KEY(), bytes32(uint256(uint160(callbackAddress))));
-    // }
+    function run(address callbackAddress) public {
+        QuarkWallet self = QuarkWallet(payable(address(this)));
+        bytes32 CALLBACK_SLOT = self.CALLBACK_SLOT();
+        bytes32 data = bytes32(uint256(uint160(callbackAddress)));
+        assembly {
+            sstore(CALLBACK_SLOT, data)
+        }
+    }
```

Comment:
Since we haven't moved to TSTORE yet for callbacks, I think it's fine to keep this for now. This test script (and it's associated test) will become obsolete once we move to TSTORE in a follow-up PR.